applications/nrf5340_audio/src/bluetooth/ble_core.c

```diff
@@ -40,6 +40,7 @@ static void net_core_timeout_handler(struct k_timer *timer_id)
 	ERR_CHK_MSG(-EIO, "No response from NET core, check if NET core is programmed");
 }
 
+#if !CONFIG_BT_PRIVACY
```

Comment:
```suggestion
#if !defined(CONFIG_BT_PRIVACY)
```

Same on line 191
I see that we mostly use only parenthesis, and sometimes "defined".
We should have a task for deciding one way of doing it and then make sure every #if is the same.

Reply:
Added paranthesis